src/tests/conformance/common.rs

```diff
@@ -0,0 +1,15 @@
+use crate::{setup::node::Node, tools::synth_node::SyntheticNode};
+
+pub async fn start_synth_node() -> SyntheticNode {
+    let node_config = pea2pea::Config {
+        listener_ip: Some("127.0.0.1".parse().unwrap()),
+        ..Default::default()
+    };
+    SyntheticNode::new(node_config).await
+}
+
+pub async fn start_ripple_node() -> Node {
```

Comment:
Note: it's fine here for now but if we use this often enough we may want it implemented on `Node` itself.

Reply:
same for `SyntheticNode` tbh

Reply:
Will do.